api/src/main/java/org/apache/brooklyn/api/entity/Entity.java

```diff
@@ -314,4 +361,80 @@
          */
         <T> void emit(Sensor<T> sensor, T value);
     }
+    
+    @Beta
+    public interface PolicySupport {
+        /**
+         * @return an immutable thread-safe view of the policies.
+         */
+        Collection<Policy> getPolicies();
```

Comment:
The other benefit is that it can be used in the enhanced for loop, e.g. `for(Policy p : policies())`.  The implementation could just return the `Iterator` from the `Collection` that is currently constructed for `getPolicies()`